frontend/src/components/document/DocumentView/StyledText.jsx

```diff
@@ -1,32 +1,22 @@
 import { getCommentThreadsOnTextNode } from "@/util/editorCommentUtils";
+import { getRedactionsOnTextNode } from "@/util/editorRedactionUtils";
 import CommentedText from "./CommentedText";
+import RedactedText from "../Redactions/SuggestedRedaction";
+import RedactionPopover from "../Redactions/RedactionPopover";
+import {useState, useEffect, React} from "react";
 
 export default function StyledText({ attributes, children, leaf }) {
 
-  //TO DO: trim space off of highlight
```

Comment:
Are these supposed to be removed? merge conflict? They seem kinda important lol

Reply:
Yup, thanks for the catch